src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -83,7 +83,7 @@ class EventOrGroupHeader extends React.Component {
 
     return (
       <div className={cx}>
-        <h3 className="truncate">{this.getTitle()}</h3>
+        <h3 className="truncate"> {this.getTitle()}</h3>
```

Comment:
Is this extra space intentional? It seems to be causing a change in this file as well as a number of snapshots